i18n/translation-en.json

```diff
@@ -52,6 +52,7 @@
     "callout": "Callout",
     "crop": "Crop Page",
     "caret": "Caret",
+    "countMeasurement": "Count measurement",
```

Comment:
Can you follow [this wiki](https://github.com/XodoDocs/webviewer/wiki/WebViewer-UI-Localization) to generate the text for other languages?